mix.exs

```diff
@@ -46,7 +46,6 @@ defmodule Surgex.Mixfile do
 
   defp aliases do
     [
-      check: check_alias(),
```

Comment:
Isn't this being used, or helpful during development?

Reply:
We used to use that. It was an check on CI, but we moved onto the `fersha-elixir-orb` and now we [have job called `static-check`](https://app.circleci.com/pipelines/github/surgeventures/surgex/78/workflows/6f982968-bd84-4595-8257-2ddd18fa2c13/jobs/180) which do does those things as "normal" separate commands.